pages/homepage.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <div class="bg-grey-400 w-full">
+    <div class="home-container md:w-75 m-auto bg-white">
+      <div class="hidden md:block">
+        <Nav/>
+      </div>
+      <div class="ml-8 md:hidden">
+          <SearchBar/>
+      </div>
+
+      <section class="banner-section bg-grey-300 grid md:grid-cols-2 px-20 py-6">
+        <div class="inline-block w-11/12 md:w-9/12 mt-8">
+          <h1 class="font-bold text-4xl">Autolizing</h1>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sit amet tincidunt libero. Duis.</p>
+          <button class="btn btn-purple mt-4">Search product</button>
+        </div>
+        <div class="hidden md:inline-block">
+          <img src="../assets/img/audiQ5.png" alt="audiQ5" class="audi w-4/5 h-auto">
+        </div>
+      </section>
+
+      <div class="px-0">
+        <div class="bg-white pt-8 px-4">
+
+            <div class="main flex flex-wrap md:ml-4 md:flex-no-wrap flex-col-reverse md:flex-row">
+              <div class="parent1 flex flex-col flex-no-wrap">
+                <div class="side-title flex mb-4">
+                  <div class="bg-accent-purple w-1 h-8"></div>
+                  <h1 class="text-accent-purple font-bold ml-4">NEW LISTINGS</h1>
+                </div>
+                <SideProductCard v-for="(sideItem, key) in sideItems" :key="key" :sideItem="sideItem"/>
+              </div>
+              <div class="parent2 flex flex-row flex-wrap md:mt-12">
+                <ProductCard v-for="(mainItem, key) in mainItems" :key="key" :mainItem="mainItem"/>
+              </div>
+            </div>
+
+        </div>
+      </div>
+
+      <div>
+        <Footer/>
+      </div>
+
+    </div>
+  </div>
+
+</template>
+
+<script>
+  import Nav from '~/layouts/nav.vue';
+  import Footer from '~/layouts/footer.vue';
+  import SearchBar from '~/components/SearchBar.vue';
+  import ProductCard from '~/components/GoodsCard.vue';
+  import SideProductCard from '~/components/SideGoodsCard.vue';
+  import AudiQ5 from '~/assets/img/audiQ5.png';
+
+    export default {
+      components: {Nav, Footer, SearchBar, AudiQ5, ProductCard, SideProductCard},
+      name: "homepage",
+      data() {
+        return {
+          sideItems:[
```

Comment:
Please, move it to a separate file in folder assets/data and then import and use this data.

Reply:
sideItems